pkg/kubelet/eviction/helpers_test.go

```diff
@@ -105,9 +114,25 @@ func TestParseThresholdConfig(t *testing.T) {
 			expectErr:               true,
 			expectThresholds:        []Threshold{},
 		},
+		"neg-reclaim": {
+			evictionHard:            "",
+			evictionSoft:            "",
+			evictionSoftGracePeriod: "",
+			evictionMinReclaim:      "memory.available=-300Mi",
+			expectErr:               true,
+			expectThresholds:        []Threshold{},
+		},
+		"duplicate-reclaim": {
+			evictionHard:            "",
+			evictionSoft:            "",
+			evictionSoftGracePeriod: "",
+			evictionMinReclaim:      "memory.available=-300Mi,memory.available=-100Mi",
```

Comment:
good point, will add and then self tag.  it was missed when i didn't bring over the disk stuff.